src/MetaMorpho.sol

```diff
@@ -153,12 +153,9 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// - there's no pending value;
     /// - the timelock has not elapsed since the pending value has been submitted;
     /// - the timelock has expired since the pending value has been submitted.
-    modifier withinTimelockWindow(uint256 submittedAt) {
+    modifier afterTimelock(uint256 submittedAt) {
```

Comment:
We should also remove the comment just above this line about the timelock expiration

Reply:
Good catch!